Gemfile

```diff
@@ -11,6 +11,8 @@ gem "flutie"
 gem "high_voltage"
 gem "i18n-tasks"
 gem "jquery-rails"
+gem 'jbuilder'
+gem 'oj'
 gem "neat", "~> 1.5.1"
```

Comment:
Also, we use single quotes. I know this is how suspenders set it up, but it's a pretty easy find and replace when you find double quotes. Install rubocop for atom and it will catch these super easily.